examples/utils/import-testing/src/testExports.ts

```diff
@@ -77,6 +81,11 @@ export class Canvas extends schema.object("Canvas", { stuff: [NodeMap, NodeList]
 export const POJO = schema.object("POJO", { stuff: [NodeMap, NodeList] });
 export type POJO = NodeFromSchema<typeof POJO>;
 
+export const POJOAlpha = schemaAlpha.object("POJO", { stuff: [NodeMap, NodeList] });
+{
+	type _check = requireAssignableTo<typeof POJOAlpha, ObjectNodeSchema>;
```

Comment:
Since no API-Report was produced for this change, it seemed possible that the generated d.ts files either were not impacted by this bug, or were not fixed by it.

Thus I replicated the test here, and confirmed this failed to build before this fix, and works after it. Therefore, the alpha API of the package has changed, but not API report delta was produced.